internal/server/http/handler_test.go

```diff
@@ -69,6 +71,59 @@ func TestImposterHandler(t *testing.T) {
 	}
 }
 
+func TestImposterHandler_Variables(t *testing.T) {
+	var headers = make(map[string]string)
+	headers["Content-Type"] = "application/json"
+
+	responseId1 := "test/testdata/imposters_variables/responses/gopher_1_response.json"
+	responseId2 := "test/testdata/imposters_variables/responses/gopher_2_response.json"
+	responseId1Variable1 := "test/testdata/imposters_variables/responses/gopher_1_1_response.json"
+	responseId1Variable2 := "test/testdata/imposters_variables/responses/gopher_1_2_response.json"
+
+	imposterFilePath := "test/testdata/imposters_variables/gopher_variables.imp.json"
+	imposterFile, _ := os.Open(imposterFilePath)
+	defer imposterFile.Close()
+	imposterBytes, _ := io.ReadAll(imposterFile)
+
+	var imposters []Imposter
+	err := json.Unmarshal(imposterBytes, &imposters)
+	assert.NoError(t, err)
```

Comment:
```suggestion
	require.NoError(t, err)
```

The recommendation for error-related assertions is `require`.